### ISSUE TITLE
Wishlist: disable and enable ligatures inside the document

### ISSUE THREAD

**Issue by AlMa1r** [schlcht/microtype#34]
Continuing http://tex.stackexchange.com/questions/569282, running `pdflatex` on
```latex
\documentclass{standalone}
\usepackage[T1]{fontenc}
\usepackage{babel}
\usepackage{newtx}
\begin{document}
bijection \foreignlanguage{dutch}{Dijkstra} \foreignlanguage{french}{Dijon}
\end{document}
```
yields

> bĳection Dĳkstra Dĳon

Here, the ligature ĳ is employed in all three words (to test it, copy and past from the PDF viewer evince).
Following https://tex.stackexchange.com/a/569293 and running `pdflatex` on
```latex
\documentclass{standalone}
\usepackage[T1]{fontenc}
\usepackage{babel}
\usepackage{newtx}
\usepackage{microtype}
\DisableLigatures[i]{encoding=T1}
\begin{document}
bijection \foreignlanguage{dutch}{Dijkstra} \foreignlanguage{french}{Dijon}
\end{document}
```
we obtain

> bijection Dijkstra Dijon

with ij as two characters (to test it, copy and past from the PDF viewer evince).
However, in certain Dutch words (Dĳkstra, vrĳgeven, …) some folks might prefer the digraph ĳ rather than the two characters ij, BUT there is no corresponding enabling command, such as `\EnableLigatures[i]{encoding=T1}`, and `\DisableLigatures` can be used only in preamble per documentation of microtype v3.1a.

Therefore, we ask for a means to selectively enable and disable ligatures as we go. Future usage example: `\addto\extrasdutch{\EnableLigature[ĳ]{encoding=T1}}` and `\addto\extrasngerman{\DisableLigature[ĳ]{encoding=T1}}`.

**Comment by hpvd** [comment]
looks like this is part of the latest release, see announcement text:

Announcement text: 
- support for tracking/letterspacing with XeTeX
-new default for letterspacing: 50
-new key `features' for \SetTracking to enable/disable fontspec
   `Ligatures' features (LuaTeX/XeTeX only)
-new values `none' and `all' for the `no ligatures' key of \SetTracking
- ... as well as the customary various fixes and improvements

https://ctan.org/ctan-ann/id/Z11L89MCwe3UbL5U@prptp

**Comment by schlcht** [comment]
@hpvd no, sorry, that's a misunderstanding: the latest release allows to en-/disable ligatures within letterspaced text by way of the "Ligatures" key that fontspec provides. There's still no possibility to enable or disable ligatures generally (via `\DisableLigatures`) within the document. This has to wait for the next release.
